packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx

```diff
@@ -77,6 +79,8 @@ export class SwatchColorPickerBase extends BaseComponent<ISwatchColorPickerProps
       getStyles,
     } = this.props;
 
+    const htmlProps = getNativeProps(this.props, htmlElementProperties, ['data']);
```

Comment:
is there an overridden data prop on SwatchColorPicker? If not, then you don't need the third param

Reply:
Do we even need to get the native props here? Couldn't we just spread the props we were given until we get into Grid and then get the nativeProps. Note: you'd want to make sure any of the props that map to native html attributes we want overriden or that we are changing props as part of the creation of the Grid in the SwatchColorPicker to be defined after the spread (e.g. items, onBlur, and theme for example)

Reply:
Got it, there were no particular reason for this. I made an assumption that it was needed based off the ResizeGroup PR, but you're right, it's not needed.